torch/hub.py

```diff
@@ -209,6 +212,58 @@ def _get_cache_or_reload(github, force_reload, verbose=True, skip_validation=Fal
 
     return repo_dir
 
+def _get_trusted_input(repo):
+    # separate function for testing
+    response = input(
+        f"The repository {repo} does not belong to the list of trusted repositories and as such cannot be downloaded. "
+        "Do you trust this repository and wish to add it to the trusted list of repositories (y/N)?")
+    return response
+
+def _check_repo(repo, trust_repo=None, calling_fn="load"):
```

Comment:
Similarly to comment above

```suggestion
def _check_repo(repo, trust_repo, calling_fn):
```